governance/default.go

```diff
@@ -667,18 +667,26 @@ func (g *Governance) initializeCache() error {
 		}
 	}
 
-	lastGovernanceBlock, lastGovernanceStateBlock := indices[len(indices)-1], atomic.LoadUint64(&g.lastGovernanceStateBlock)
-	if lastGovernanceBlock >= lastGovernanceStateBlock {
-		// head block number is used to get the appropriate g.currentSet and g.actualGovernanceBlock
-		headBlockNumber := uint64(0)
-		if headBlockHash := g.db.ReadHeadBlockHash(); !common.EmptyHash(headBlockHash) {
-			if num := g.db.ReadHeaderNumber(headBlockHash); num != nil {
-				headBlockNumber = *num
-			}
+	// head block number is used to get the appropriate g.currentSet and g.actualGovernanceBlock
+	headBlockNumber := uint64(0)
+	if headBlockHash := g.db.ReadHeadBlockHash(); !common.EmptyHash(headBlockHash) {
+		if num := g.db.ReadHeaderNumber(headBlockHash); num != nil {
+			headBlockNumber = *num
+		}
+	}
+	newBlockNumber, newGovernanceSet, _ := g.ReadGovernance(headBlockNumber)
```

Comment:
@yoomee1313 How about adding a comment about the point that we were confused? (e.g. difference from `g.db.ReadGovernance`, or why we put this out of the above for loop).

Don't we need to handle the returned error?

Reply:
@jeongkyun-oh Thanks, I added error handling logic and some comments to clarify the function's behavior. Please take a look! https://github.com/klaytn/klaytn/pull/1168/commits/55ad79072d91478eaae2f86e46664ebfa6c4b0d0